src/components/checkbox/checkbox.stories.mdx

```diff
@@ -24,18 +31,14 @@ import { Checkbox, CheckboxGroup } from ".";
 import { Checkbox } from "carbon-react/lib/components/checkbox";
 ```
 
-## Designer Notes
-
-- Disabled or read-only checkboxes might be difficult for a user to distinguish visually, so try to avoid this.
-- Consider ‘smart default’ selections, based on what your user is likely to choose. But, users may well leave the defaults in place, so make sure any consequences are easy to undo, and not harmful.
-- Rather than a checkbox to accept legal terms, consider labelling a button ‘Accept Terms and Continue’ instead.
-
 ## Related Components
 
 - Choosing one option from a longer list? [Try Radio Button](/?path=/docs/experimental-radiobutton--default "Radio Button").
 - Choosing one option from a very long list? [Try Select](?path=/docs/select--default-story "Select").
 - Choosing one option from a highly visible range? [Try Button Toggle](/?path=/story/button-toggle--default "Button Toggle").
 
+Please Note: Carbon Library do offer the functionality for `Disabled` checkboxes, but Design System documentation do not recommend using those.
+
```

Comment:
Thanks for adding this in @DobroTora 👍🏼  What do you think about moving this disclaimer to the `Disabled` section on line 79, similarly as you have done for `RadioButton` so it reads a little easier?

Reply:
Sure.